firewalld/files/zone.xml

```diff
@@ -64,6 +72,14 @@
   <forward-port port="{{ v.portid }}" protocol="{{ v.protocol }}"{%- if 'to_port' in v %} to-port="{{ v.to_port }}"{%- endif %}{%- if 'to_addr' in v %} to-addr="{{ v.to_addr }}"{%- endif %} />
   {%- endfor %}
 {%- endif %}
+{%- if 'source_ports' in zone %}
+  {%- for v in zone.source_ports %}
+    {%- if 'comment' in v %}
+  <!-- {{ v.comment }} -->
+    {%- endif %}
+  <source-port port="{{ v.port }}" protocol="{{ v.protocol }}"/>
```

Comment:
Space before the closing tag, i.e. `"/>` => `" />`.

Reply:
should be fixed